cmd/kubeadm/app/phases/certs/renewal/manager.go

```diff
@@ -271,14 +299,17 @@ func (rm *Manager) CreateRenewCSR(name, outdir string) error {
 	return nil
 }
 
-// GetExpirationInfo returns certificate expiration info.
+// GetExpirationInfo returns certificate and related certificate authority expiration info.
 // For PKI certificates, use the name defined in the certsphase package, while for certificates
 // embedded in the kubeConfig files, use the kubeConfig file name defined in the kubeadm constants package.
 // If you use the CertificateRenewHandler returned by Certificates func, handler.Name already contains the right value.
 func (rm *Manager) GetExpirationInfo(name string) (*ExpirationInfo, error) {
 	handler, ok := rm.certificates[name]
 	if !ok {
-		return nil, errors.Errorf("%s is not a known certificate", name)
+		handler, ok = rm.certificateAuthorities[name]
```

Comment:
Same here, I think we should keep CA and Certificates management splitted vs using tricks like falling back from a map to another